src/components/Graph/Graph.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Node from './Node';
+import Edge from './Edge';
+
+const ListItem = ({ items, renderItem }) => {
+  return items.map((item, index) => renderItem(item, index));
+};
```

Comment:
You call this ListItem but this is the entire list. 
If you want you can create a generic list it should expect a component to render and data to map over